StudentDetails.java

```diff
@@ -0,0 +1,155 @@
+import java.util.*;
+import java.util.Scanner;
+
+class StudentData {
```

Comment:
Why usage of a default access modifier? Also, the suffix __Data__ adds no meaning to the naming of the class at all. Just use __Student__ instead of __StudentData__. Avoid suffixes that add no meaning.